src/main.py

```diff
@@ -458,6 +608,32 @@ async def knative_request(
             )
 
 
+async def redis_stream_request(
+    redis_client: redis.Redis,
+    redis_stream: str,
+    body: dict[str, typing.Any],
+) -> None:
+    """Makes redis stream request.
+
+    Parameters
+    ----------
+    in_process_requests_gauge : `prometheus_client.Gauge`
+        A gauge to be updated with the start and end of the request.
+    client : `redis.Redis`
+        The redis stream client.
+    redis_stream : `string`
+        The name of the redis stream.
+    body : `dict[str, typing.Any]`
```

Comment:
This will try to link to a type literally called `dict[str, typing.Any]`; see examples in the [DM style guide](https://developer.lsst.io/python/numpydoc.html#py-docstring-parameter-types-dict):
```suggestion
    body : `dict` [`str`, `typing.Any`]
```